common/lib/xmodule/xmodule/x_module.py

```diff
@@ -27,14 +27,14 @@
 
 from xblock.fragment import Fragment
 from xblock.runtime import Runtime, IdReader, IdGenerator
```

Comment:
Nit: xblock should be its own section or ordered?  If its own section, should opaque_keys join it?